data/containers/autoyast_containers.xml.ep

```diff
@@ -3,14 +3,14 @@
 <profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
   <suse_register>
     <do_registration config:type="boolean">true</do_registration>
-    % unless ($check_var->('VERSION', '15-SP4')) {
+    % unless ($check_var->('VERSION', '15-SP6')) {
     <install_updates config:type="boolean">true</install_updates>
```

Comment:
I think this can be dropped as we have also an `install_update` openqa module after the host is deployed